geomesa-core/src/main/scala/org/locationtech/geomesa/core/data/AccumuloDataStore.scala

```diff
@@ -120,9 +121,8 @@ class AccumuloDataStore(val connector: Connector,
   private val visibilityCheckCache = new mutable.HashMap[(String, String), Boolean]()
                                          with mutable.SynchronizedMap[(String, String), Boolean]
 
-  // TODO memory should be configurable
   private val defaultBWConfig =
-    new BatchWriterConfig().setMaxMemory(10000L).setMaxWriteThreads(writeThreads)
+    GeoMesaBatchWriterConfig().setMaxMemory(10000L).setMaxWriteThreads(writeThreads)
```

Comment:
Should we not override the memory here? Makes it so the command line params get ignored...
Overriding in StatWriter and MetaData seems reasonable as those should have smaller memory requirements.